modules/util/JitsiMeetInMemoryLogStorage.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Implements in memory logs storage, used for testing/debugging.
+ */
+export default class JitsiMeetInMemoryLogStorage {
+
+    /**
+     * Creates new <tt>JitsiMeetInMemoryLogStorage</tt>
+     */
+    constructor() {
+        /**
+         * Array of the log entries to keep.
+         * @type {array}
+         */
+        this.logs = [];
+    }
+
+    /**
+     * @return {boolean} <tt>true</tt> when this storage is ready or
```

Comment:
hmm there are no lint warning here ? it would complain that @returns must be used instead of @return